beacon_node/eth1/src/service.rs

```diff
@@ -720,13 +723,23 @@ impl Service {
 
         let mut interval = interval_at(Instant::now(), update_interval);
 
+        let num_fallbacks = self.config().endpoints.len() - 1;
         let update_future = async move {
             loop {
                 interval.tick().await;
                 self.do_update(update_interval).await.ok();
             }
         };
 
+        // Set the number of configured eth1 servers
+        metrics::set_gauge(&metrics::ETH1_FALLBACK_CONFIGURED, num_fallbacks as i64);
+        // Since we lazily update eth1 fallbacks, it's not possible to know connection status of fallback.
+        // Hence, we set it to 1 if we have atleast one configured fallback.
+        if num_fallbacks > 0 {
+            metrics::set_gauge(&metrics::ETH1_FALLBACK_CONNECTED, 1);
+        } else {
+            metrics::set_gauge(&metrics::ETH1_FALLBACK_CONNECTED, 0);
```

Comment:
This is tricky because we don't know which node is down. Could be the primary node here as well.
`ETH1_FALLBACK_CONNECTED` right now checks only if the fallback is configured. Have added a comment here
 https://github.com/sigp/lighthouse/blob/845f22f71ef9cf1a057517d7518b919f76345836/beacon_node/eth1/src/metrics.rs#L42-L45